src/core/Classes/Post_Editor.php

```diff
@@ -408,8 +422,12 @@ public static function action_save_post_authors_metabox($post_id, $post)
         $authors = self::remove_dirty_authors_from_authors_arr($authors);
 
         $fallbackUserId = isset($_POST['fallback_author_user']) ? (int)$_POST['fallback_author_user'] : null;
+        $disableAuthorBox = isset($_POST['ppma_disable_author_box']) ? (int)$_POST['ppma_disable_author_box'] : 0;
 
         Utils::set_post_authors($post_id, $authors, true, $fallbackUserId);
+        if (isset($_POST['ppma_save_disable_author_box']) && (int)$_POST['ppma_save_disable_author_box'] > 0) {
```

Comment:
@ojopaul here you could simplify this conditional using the var `$disableAuthorBox`.

Reply:
I think you guys can handle this one without me ;)

Reply:
@andergmartins `$_POST['ppma_save_disable_author_box']` is not same as `$disableAuthorBox = isset($_POST['ppma_disable_author_box']) ? (int)$_POST['ppma_disable_author_box'] : 0;` To prevent the update from running when not intended for example, in bulk edit, an hidden `ppma_save_disable_author_box` field was added which is different from `ppma_disable_author_box` that's been defined with `$disableAuthorBox`  and the hidden field is always checked. That way, we'll know the submission was made from a page where the metabox is present.

The reason is because the input type is a checkbox which shows $disableAuthorBox will be 0 when the filter run through bulk edit action that doesn't have the the checkbox in post thereby reseting whatever option was saved.

Or do you mean something else?

Reply:
@ojopaul oh right, my mistake. I missed that detail. Looks good to me 👍🏻  tks